src/python/CMakeLists.txt

```diff
@@ -290,8 +291,13 @@ if(PYTHONINTERP_FOUND)
     # Other .py files
     file(COPY "gudhi/persistence_graphical_tools.py" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/gudhi")
     file(COPY "gudhi/representations" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/gudhi/")
+<<<<<<< HEAD
```

Comment:
You committed the merge without fixing the conflict, the result is clearly not going to work...

Reply:
My bad. It should be better now.